examples/chef/chef_util.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+import json
+import os
+import hashlib
+
+
+ci_allowlist = ['lighting-app.zap']
+cd_platforms_meta = {
```

Comment:
Although the proposed solution is elegant, I would prefer if configs lived as data and not code. I put the config in a json file and deserialize. It could be changed to YAML in the future for consistency.

Reply:
Noting that allowlist is still in code.